bsonproto/objectid.go

```diff
@@ -1,3 +1,33 @@
 package bsonproto
 
-type ObjectID struct{}
+import "fmt"
+
+// ObjectID represents BSON scalar type ObjectID.
+type ObjectID [12]byte
+
+// SizeObjectID is a size of the encoding of [ObjectID] in bytes - 12.
+const SizeObjectID = 12
+
+// EncodeObjectID encodes [ObjectID] value v into b.
+//
+// b must be at least 12 bytes long; otherwise, EncodeObjectID will panic.
+// Only b[0:12] bytes are modified.
+func EncodeObjectID(b []byte, v ObjectID) {
+	_ = b[11]
```

Comment:
Looks like this is the only place where we do such assert. Not a big deal but should we add in other places? or change next line to `copy(b[:12], v[:])`, dunno.

Reply:
Functions like `binary.LittleEndian.Uint64` do that under the hood, too

Reply:
I know but do we need that?

Reply:
LOL, anyway.